JourneysSquad/JourneysSquad/View/BillListView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+/// A view displaying the list of bills.
+struct BillListView: View {
+    // MARK: - Internal interface
+
+    var body: some View {
+        List {
+            ForEach(BillListDummyData.listData, id: \.headerText) { section in
+                Section(header: Text(section.headerText).font(.system(size: fontBodySize))) {
```

Comment:
Maybe it would be more readable ?

```suggestion
                Section(header: Text(section.headerText)
                .font(.system(size: fontBodySize))) {
```

Reply:
Fixed